### PR TITLE
Hello microverse pull request

### PR DESCRIPTION
**Pull request by RichardKaranuMbuti**
In this pull request I;
  added index.html file
   style.css file

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @RichardKaranuMbuti, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- HTML and CSS file ✔️ 
- .gitIgnore file ✔️
- Commit message ✔️

## Required Changes ♻️
_Check the comments under the review._
- [x] I would advise you to add a linter so that your code can be guided by best practices. You can find the linter setup [here](https://github.com/microverseinc/linters-config)
- [ ] I would advise you to use a well-detailed readme file template provided by Microverse, it would help you and the visitors to understand what the repo entails at a glance. You can copy the file [here](https://github.com/microverseinc/readme-template) and adapt it to your own.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RichardKaranuMbuti** [comment]
Thank you for the review.I added the linters file in the .github/workflows
folder as it was explained to me. Would you please make it clear what you
mean by I should include linters.
Thank you

**Comment by RichardKaranuMbuti** [comment]
added linters
edited readme to follow microverse template

File: README.md [hunk 1]
```diff
@@ -1 +1,287 @@
...
+
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
```
Comment (Felix45) [inline README.md:117]: - [ Optional ] The `Getting Started` section of your readme file is very important, this means you always have to fill it correctly. Please see an example of how I would fill in this section in my `readme` file, the instructions you give in the `readme` file should always be clear and easy for someone to follow.



## 💻 Getting Started <a name="getting-started"></a>

To get a local copy up and running, follow these steps.

### Prerequisites

In order to run this project you need:
 - NodeJS
 - Browser

### Setup

Clone this repository to your desired folder:



```sh
  git clone git@github.com:RichardKaranuMbuti/Hello-microverse.git
  cd Hello-microverse
```

### Install

Install project dependencies in this project with:

```sh
  npm install
```

### Usage

To run the project, open the project with live server and visit [http://localhost:5500](http://localhost:5500)

### Open in browser

http://localhost:5500

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @RichardKaranuMbuti , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] All linter checks are set up and passing with green check marks :heavy_check_mark: 
- [ ] GitHub flow is correctly used :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**